access/grpc/convert_test.go

```diff
@@ -202,6 +202,8 @@ func TestConvert_Event(t *testing.T) {
 	eventB, err := messageToEvent(msg, nil)
 	require.NoError(t, err)
 
+	_ = eventB.Value.Type().ID()
```

Comment:
what does this do?

Reply:
Good question, I'll add comments

Reply:
Added in 21631e000c73baca6c87422d031a7c4e38f6275d